src/highlight/HighlightAnnotations.tsx

```diff
@@ -92,7 +94,7 @@ const HighlightAnnotations = (props: Props): JSX.Element => {
     }, [isSelecting]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
It looks like `isPending` needs to be added to the deps array for both effects. We should remove these `eslint-disable-line` directives to avoid bugs.